includes/class-wcmp-data.php

```diff
@@ -22,13 +22,13 @@ class WCMP_Data
      */
     public const CARRIERS_HUMAN = [
         DPDConsignment::CARRIER_NAME   => 'DPD',
```

Comment:
moet dpd niet weg?

Reply:
Nee die laten we staan alle opties van DPD gaan we verbergen.